### ISSUE TITLE
fatal error: AsyncTCP.h: No such file or directory

### ISSUE THREAD

**Issue by ctvedt** [tube0013/esphome-stream-server-v2#6]
Not able to update my esphome stream server from 2023.4.4 due to the error below. Have tried to clean build files.

```
Compiling /data/esp32_han/.pioenvs/esp32_han/src/esphome/components/stream_server/binary_sensor.cpp.o
In file included from src/esphome/components/stream_server/binary_sensor.h:21,
                 from src/esphome/components/stream_server/binary_sensor.cpp:17:
src/esphome/components/stream_server/stream_server.h:28:10: fatal error: AsyncTCP.h: No such file or directory

******************************************************************
* Looking for AsyncTCP.h dependency? Check our library registry!
*
* CLI  > platformio lib search "header:AsyncTCP.h"
* Web  > https://registry.platformio.org/search?q=header:AsyncTCP.h
*
******************************************************************

 #include <AsyncTCP.h>
          ^~~~~~~~~~~~
compilation terminated.
*** [/data/esp32_han/.pioenvs/esp32_han/src/esphome/components/stream_server/binary_sensor.cpp.o] Error 1
========================= [FAILED] Took 21.54 seconds =========================
```

**Comment by ctvedt** [comment]
See PR #7

**Comment by ptr727** [comment]
Same issue here, please merge the PR.

**Comment by ptr727** [comment]
Fixed after PR #7 was merged, thank you.
